setup.py

```diff
@@ -39,7 +39,12 @@
         "Programming Language :: Python :: 3.7",
     ],
     description="FV3Config is used to configure and manipulate run directories for FV3GFS.",
-    entry_points={"console_scripts": ["fv3run=fv3config.fv3run.__main__:main"]},
+    entry_points={
+        "console_scripts": [
+            "fv3run=fv3config.fv3run.__main__:main",
+            "write_run_directory=fv3config.cli:write_run_directory",
```

Comment:
Is this a specific-enough name for a bash command? I'm slightly worried about name clashes, but I'm OK with the name as-is. Maybe `write_fv3_directory` or `fv3dir`?

Reply:
I think it's okay for now. We can always change it if somebody reports a name clash.

Reply:
Yeah, [probably not an issue](https://www.google.com/search?q=write_run_directory).